layers/eight_mile/calibration/calibration.py

```diff
@@ -0,0 +1,49 @@
+from collections import namedtuple
+import numpy as np
+
+__all__ = ["calibration_bins", "multiclass_calibration_bins", "binary_calibration_bins", "average_confidence", "Bins"]
+
+
+Bins = namedtuple("Bins", "accs confs counts edges")
+
+
+def multiclass_calibration_bins(true: np.ndarray, probs: np.ndarray, bins: int) -> Bins:
+    """
+    """
+    preds = np.argmax(probs, axis=1)
+    pred_probs = np.max(probs, axis=1)
+    return binary_calibration_bins(true == preds, pred_probs, bins=bins)
+
+
+calibration_bins = multiclass_calibration_bins
+
+
+def binary_calibration_bins(true: np.ndarray, probs: np.ndarray, bins: int) -> Bins:
```

Comment:
correct?

Reply:
That makes more sense. I'll make that change soon

Reply:
Updated